peodd/peodd.py

```diff
@@ -28,20 +28,21 @@
 
 import click
 import tomli
-
 from release_tools.project import Project
-from release_tools.semverup import find_pyproject_file
 from release_tools.repo import RepositoryError
+from release_tools.semverup import find_pyproject_file
 
 VERSION_NUMBER_REGEX = r"\d+(?:\.\d+)+"
 
 
 @click.command()
 @click.option('-o', '--output',
-              default="requirements-dev.txt",
-              show_default=True,
-              help="Output file for the dependencies")
-def main(output):
+              help="Output filename for the dependencies")
+@click.option('--non-dev',
+              default=False,
+              is_flag=True,
```

Comment:
I think you deleted this line in the last commit. I would suggest to remove it for `output` :ballot_box_with_check: but keep it for `non-dev`
```
              show_default=True,
```

Reply:
from my last commit:

```python
@click.command()
@click.option('-o', '--output',
              help="Output filename for the dependencies")
@click.option('--non-dev',
              default=False,
              is_flag=True,
              help="Output non-dev dependencies")
```

Shouldn't it be `False`since we want to specifically use `--non-dev` when outputting the non-dev dependencies?

Reply:
Yes, you are right. :100: 
Sorry for the wrong tip.

Reply:
Yes, `default=False` and `show_default=True`. It basically shows that default is set to False in the help manual.